src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java

```diff
@@ -100,6 +101,7 @@
             MEMBER_DAYS_REPORT, RESPONDENTS_REPORT, SESSION_DAYS_REPORT, ECC_REPORT);
     private static final List<String> SCOTLAND_HEARING_LIST = List.of("Reading Day", "Deliberation Day",
             "Members meeting", "In Chambers");
+    private static final DateTimeFormatter dateTimePattern = DateTimeFormatter.ofPattern("dd MMMM yyyy");
```

Comment:
Should Month format be MMM as there are months which have less than 4 characters?

Reply:
It shouldn't affect May. MMMM will still show May. The original was "d MMMM yyyy". I have just changed it to "dd" to allow it to be in order. It was already MMMM